php-plugin/src/test/resources/checks/AvoidDoubleQuote.php

```diff
@@ -31,5 +21,3 @@
   myFunction('name', 'age', 'isStudent');
 
   myFunction("name", 'age', "isStudent"); // NOK {{Avoid using double quote ("), prefer using simple quote (')}}
-
-?>
```

Comment:
Can you add a blank line at the end of files? To comply with the standard on this point too
Check the PSR-12 here: https://www.php-fig.org/psr/psr-12/#22-files